crates/backend/src/codegen.rs

```diff
@@ -624,17 +624,26 @@ impl ToTokens for ast::ImportEnum {
 
         (quote! {
             #[allow(bad_style)]
-            #[derive(Copy, Clone, Debug)]
+            #[derive(Copy, Clone, PartialEq, Debug)]
```

Comment:
I've just realized (after the fact, sorry!) that in other places in wasm-bindgen we don't explicitly list derive modes but rather pass through the original annotations. Here there's no `#[wasm_bindgen]` equivalent for generating an imported `enum` so it doesn't matter too too much, but long-term we may want this line to be `#(#rust_attrs)*` and store attributes in the `ImportEnum`

Reply:
:+1: